docs/alerts.md

```diff
@@ -91,7 +245,41 @@ The elasticsearch Cluster is predicted to be out of disk space within the next 6
 
 ### Troubleshooting
 
-Check the disk space of the Elasticsearch node.
+1. Check the disk space of the Elasticsearch node:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_nodes/stats/fs?pretty
+   ```
+   Check the “nodes.node_name.fs” field to determine the free disk space on that node.
+
+2. The prediction is done based on the current trend in occupying the disk space on that node.
+
+3. If it looks like the disk space will not get over then
+    - Ignore the alert.
```

Comment:
So, this alert is based on a linear trend... if it fires its because it will eventually run out in 6 hours based on the rate at which it is being filled up. I'm not sure point 3 here is a proper suggestion...

Reply:
Removed it.